test/clj_kondo/main_test.clj

```diff
@@ -2082,16 +2082,16 @@
     (when (.exists (io/file ".clj-kondo"))
       (rename-path ".clj-kondo" ".clj-kondo.bak"))
     (make-dirs ".clj-kondo")
-    (lint! "(ns app.core) (defn foo [])" "--cache")
+    (lint! "(ns app.core) (defn foo [])" "--cache" "true")
```

Comment:
Can you revert this? `--cache` is a boolean flag: if it's present, it's implicitly true.